pulse/pulse/templates/includes/https/header.html

```diff
@@ -1,6 +1,6 @@
 <div class="container mx-auto items-center sm:w-4/5 xl:w-3/5">
 	<h1 class="text-4xl sm:text-5xl">HTTPS</h1>
-	<h2 class="mb-4 sm:mb-6 text-xl sm:text-2xl">Last updated April 16, 2018</h2>
+	<h2 class="mb-4 sm:mb-6 text-xl sm:text-2xl">Last updated {{ scan_date | date("%B %d, %Y") }}</h2>
```

Comment:
The `date(...)` thing is interesting, what do?

Reply:
Formats it pretty!

Reply:
Cool